geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientStatsDUnitTest.java

```diff
@@ -292,71 +262,60 @@ public static void checkStatistics() {
   public static void checkStatisticsWithExpectedValues(int reconnectionCount, int queueDropCount,
       int enqueueCount) {
     try {
-      Cache cache = CacheServerTestUtil.getCache();
+      Cache cache = getCache();
       org.apache.geode.LogWriter logger = cache.getLogger();
       CacheServerImpl currentServer =
-          (CacheServerImpl) (new ArrayList(cache.getCacheServers()).get(0));
+          (CacheServerImpl) (new ArrayList<>(cache.getCacheServers()).get(0));
       Acceptor ai = currentServer.getAcceptor();
       CacheClientNotifier notifier = ai.getCacheClientNotifier();
       CacheClientNotifierStats stats = notifier.getStats();
       logger.info("Stats:" + "\nDurableReconnectionCount:" + stats.get_durableReconnectionCount()
           + "\nQueueDroppedCount" + stats.get_queueDroppedCount()
           + "\nEventsEnqueuedWhileClientAwayCount" + stats.get_eventEnqueuedWhileClientAwayCount());
-      assertEquals(reconnectionCount, stats.get_durableReconnectionCount());
-      assertEquals(queueDropCount, stats.get_queueDroppedCount());
-      assertEquals(enqueueCount, stats.get_eventEnqueuedWhileClientAwayCount());
+      assertThat(stats.get_durableReconnectionCount()).isEqualTo(reconnectionCount);
```

Comment:
If these stats are internally implemented with `Statistics` type we need to `await` on the assertion because `Statistics.getX()` calls are not thread synchronized. It could take a few hits for the threads to all see the same value in a steady state.

Reply:
fixed.